.circleci/config.yml

```diff
@@ -39,7 +39,7 @@ jobs:
           key: dependencies-{{ checksum "yarn.lock" }}
       - run:
           name: Test
-          command: yarn test
+          command: yarn run test:ci
```

Comment:
The `run` keyword should be unnecessary 😁 but I also don't mind it being there, just thinking out loud.

Reply:
TIL, thanks!